### PR TITLE
[reggen] Revert version regs change

### PR DESCRIPTION
**Pull request by msfschaffner**
~~This is WIP and will need a discussion in the TC.~~
This change has been discussed in the TC - see explanation further below.

### CODE REVIEW

**Comment by luismarques** [comment]
What's the motivation for reverting the CIP block versions?

**Review by cfrantz — APPROVED** [review]

File: python-requirements.txt [hunk 1]
```diff
@@ -18,6 +18,8 @@ pytest==7.0.1
 pyyaml==6.0
 rich==13.3.5
 semantic_version==2.10.0
+pyyaml==6.0
+regs")
```
[the remark this answers is not in the document]
Comment (msfschaffner) [reply python-requirements.txt:22]: huh? looks like something went wrong when I rebased this. Let me correct that.

[the remark this answers is not in the document]
Comment (msfschaffner) [reply python-requirements.txt]: Ok this should be fixed now.

**Comment by msfschaffner** [comment]
The last force push adds back the `cip_id` and semantic version fields in the Hjson, together with validation code in the regtool.

File: util/reggen/ip_block.py [hunk 2]
```diff
@@ -19,6 +20,45 @@
 from reggen.signal import Signal
 from reggen.countermeasure import CounterMeasure
 
+# Known unique comportable IP names and associated CIP_IDs.
+KNOWN_CIP_IDS = {
```
Comment (msfschaffner) [inline util/reggen/ip_block.py]: @cfrantz given that this is a unique name to number mapping, and given that we do not expose CIP_IDs in registers anymore, do we even need the CIP_ID key in the Hjson? It almost feels like it is redundant at this point.

Comment (a-will) [reply util/reggen/ip_block.py]: I would argue that we should throw CIP IDs out (or at least not require them). Requiring IDs in an in-tree table prevents out-of-tree use of the tools.

**Pull request merged**